includes/shortcodes/class-wc-shortcode-my-account.php

```diff
@@ -379,6 +379,15 @@ public static function reset_password( $user, $new_pass ) {
 		if ( ! apply_filters( 'woocommerce_disable_password_change_notification', false ) ) {
 			wp_password_change_notification( $user );
 		}
+		/**
+		 * Fires after the user's password is reset.
+		 *
+		 * @since 4.9.0-dev
+		 *
+		 * @param WP_User $user     The user.
+		 * @param string  $new_pass New user password.
+		 */
+		do_action( 'after_password_reset', $user, $new_pass );
```

Comment:
This will duplicated the password change notification, see: https://github.com/WordPress/WordPress/blob/3716c8c20f075deb81b827d6c8c66937a25aacb5/wp-includes/default-filters.php#L467

Reply:
Why will it duplicate? This is the same hook in WP Core, but when WC is installed, WC overrides WP, and then (in WC) this hook currently does not exist. But maybe there's more to this issue? 

Password reset via my-account/lost-password does not trigger after_password_reset action #27795

Reply:
Please if you test it you'll see duplicated emails, because we are triggering `after_password_reset` in line 390, and in the default WP filters `wp_password_change_notification()` is hooked into `after_password_reset`.

So `wp_password_change_notification()` runs now in line 380 and also 390.

Reply:
Ahhh. Yes, I see. My apologies. In that case, I renamed the hook (action) to 'woocommerce_after_password_reset' (from 'after_password_reset'). It's an "extra" hook, but it sidesteps the issue you pointed out. I suppose that's better than nothing.